samples/kernel/condition_variables/condvar/src/main.c

```diff
@@ -5,7 +5,7 @@
  */
 #include <zephyr/kernel.h>
 #include <zephyr/arch/cpu.h>
-#include <zephyr/sys/arch_interface.h>
+#include <zephyr/arch/arch_interface.h>
```

Comment:
hmm, probably we do not need both of those included here.

Reply:
Oh yeah... removed.